contrib/utilities/inlet_profile_generator/inlet_profile_generator.py

```diff
@@ -0,0 +1,110 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 The Lethe Authors
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception OR LGPL-2.1-or-later
+
+import sympy as sp
+import argparse
+from math import ceil
+
+# Parse arguments
+parser = argparse.ArgumentParser(description="Generate inlet polynomial velocity profile with given flow rate")
+parser.add_argument("--polynomial-degree", type=int, default=2, help="Degree k of the polynomial shape (k>=0)")
+parser.add_argument("--shape", type=str, default="rectangle", choices=["rectangle", "circle"],
+                    help="Inlet cross-section shape")
+parser.add_argument("--flow-rate", type=float, default=1.0, help="Desired volumetric flow rate Q")
+parser.add_argument("--center", type=float, nargs="+", default=[0.0, 0.0],
+                    help="Center coordinates [x0, y0]")
+parser.add_argument("--length", type=float, nargs="+", default=[1.0, 1.0],
+                    help="Rectangle: lengths [ax, ay]. Circle: diameter [D]")
+parser.add_argument("--show", type=bool, default=False, help="Show a 1D centerline plot (rectangle only)")
```

Comment:
You don't need the show to request the user to give a value like True.
What I would suggest is instead something like:
https://github.com/chaos-polymtl/lethe/blob/master/examples/incompressible-flow/2d-lid-driven-cavity/post_process_Reynolds_400.py#L25

Where the fact that the argument is present creates an action that stores true. This way you could just write --show

Reply:
When I think about it, don't you think naming the command plot would also be better?